routing/router_test.go

```diff
@@ -2922,16 +2922,16 @@ func TestSendToRouteStructuredError(t *testing.T) {
 	}
```

Comment:
nit: there's a typo in the commit message, 

As is, this test will fail as a recent refactoring causes us to return a
`channeldb.FailureReason` error, since the newly added `handleSendError`
code path in the `SendToRoute` method will return the raw error, ~~ratehr~~ rather
than the `shardError`, which is of the expected type.